### PR TITLE
chore(app): add google beta icon

### LINKED ISSUE #25968: Debug - Add Chrome Beta Icon
## Current behavior

In `LayeredBrowserIcons` that is used on the Debug page, the icon for the regular "Chrome" browser is being used for the  "Chrome Beta" browser.  This can make the UI look like it is duplicating browsers when it should be showing a unique list.

## Desired behavior

Add the correct SVG icon for the "Chrome Beta" browser to the design system and use it in `LayeredBrowserIcons`.

For reference, the `cypress-services` repo has a PNG icon that it uses located here: https://github.com/cypress-io/cypress-services/blob/46073cb5c387af3b203404280a1664a85e233b93/frontend/packages/dashboard/src/lib/img/chrome-beta_32x32.png

### PR DESCRIPTION
**Pull request by jordanpowell88**
- Closes #25968 

### Additional details
- Currently the icon for the regular "Chrome" browser is being used for the "Chrome Beta" browser. This can make the UI look like it is duplicating browsers when it should be showing a unique list

### Steps to test
- `yarn workspace @packages/app dev`
- Launch the `LayeredBrowserIcons.cy.tsx` test
- View the last row of icons and you will see `Chrome`, `Chrome Canary` and `Chrome Beta` icons

### How has the user experience changed?
Screen Shot 2023-03-06 at 2 20 43 PM

### PR Tasks

- [ ] Have tests been added/updated?
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?

### CODE REVIEW

**Comment by warrensplayer** [comment]
@jordanpowell88 Pull in the latest from `develop` and resolve the conflicts and then we can review.  I had added that placeholder for Chrome Beta that got merged in.

File: packages/app/src/debug/LayeredBrowserIcons.cy.tsx [hunk 1]
```diff
@@ -19,12 +19,13 @@ describe('<LayeredBrowserIcons/>', () => {
...
-        <LayeredBrowserIcons browsers={['FIREFOX', 'WEBKIT', 'EDGE']}/>
-        <LayeredBrowserIcons browsers={['FIREFOX', 'EDGE', 'CHROME-CANARY', 'WEBKIT']}/>
-        <LayeredBrowserIcons browsers={['EDGE', 'CHROME', 'WEBKIT', 'ELECTRON']}/>
+        <LayeredBrowserIcons order={browsers} />
+        <LayeredBrowserIcons order={['CHROME', 'FIREFOX', 'EDGE']}/>
+        <LayeredBrowserIcons order={['CHROME-CANARY', 'WEBKIT', 'ELECTRON']} />
+        <LayeredBrowserIcons order={['FIREFOX', 'WEBKIT', 'EDGE']}/>
+        <LayeredBrowserIcons order={['FIREFOX', 'EDGE', 'CHROME-CANARY', 'WEBKIT']}/>
+        <LayeredBrowserIcons order={['EDGE', 'CHROME', 'WEBKIT', 'ELECTRON']}/>
+        <LayeredBrowserIcons order={['CHROME', 'CHROME-BETA', 'CHROME-CANARY']} />
```
Comment (marktnoonan) [inline packages/app/src/debug/LayeredBrowserIcons.cy.tsx]: Not a part of this PR, but it looks like the [screenshot didn't render](https://percy.io/cypress-io/cypress/builds/25735966/changed/1434750667?browser=chrome&browser_ids=34&subcategories=unreviewed%2Cchanges_requested&viewLayout=side-by-side&viewMode=new&width=800&widths=450%2C800) in Percy

Could we add an assertion to this test that would confirm the icons have actually rendered in the DOM before the Percy snapshot happens?

**Comment by astone123** [comment]
Seeing type errors related to the design system package. Are there some types we need to update on that side?

File: packages/app/src/debug/LayeredBrowserIcons.vue [hunk 2]
```diff
@@ -59,6 +60,7 @@ const BROWSER_MAP: Record<BrowserType, any> = {
   'EDGE': IconBrowserEdge,
   'WEBKIT': IconBrowserWebkit,
   'ELECTRON': IconBrowserElectronLight,
+  'CHROME-BETA': IconBrowserChromeBeta,
```
Comment (warrensplayer) [inline packages/app/src/debug/LayeredBrowserIcons.vue]: This actually needs to be `CHROME BETA` with a space and not a dash to match what is being sent from the Cloud.   Unfortunately, there is no enum value for these at this time.  You can also remove the existing row on line 56 which would be a duplicate.

Comment (warrensplayer) [reply packages/app/src/debug/LayeredBrowserIcons.vue]: Also, I found the code where the Cloud is mapping browser names to icons.  It is here: https://github.com/cypress-io/cypress-services/blob/46073cb5c387af3b203404280a1664a85e233b93/packages/common/src/logos/getLogoPath.ts

Can you add a comment here with a link to this code for future reference.

**Comment by warrensplayer** [comment]
> Seeing type errors related to the design system package. Are there some types we need to update on that side?

@astone123 Can you comment with the errors you are seeing?

**Comment by astone123** [comment]
@warrensplayer the same ones that are in CI https://app.circleci.com/pipelines/github/cypress-io/cypress/50189/workflows/cb2c9a99-7275-4146-99d3-1d1321367513/jobs/2083762

**Review by astone123 — APPROVED** [review]

**Review by warrensplayer — APPROVED** [review]
Added an assertion to the test to make sure Percy is not flaky as well as ensure that the SVGs are all there.

**Pull request merged**